doc/architecture/timestamps_overview.adoc

```diff
@@ -0,0 +1,67 @@
+ifndef::include-only-once[]
+:root-path: ../
+include::{root-path}_config.adoc[]
+endif::[]
+= Timestamps
+
+== Simulation time
+
+`Simulation time` refers to the timestamp within a simulation environment.
+
+**Definition of zero time of simulation time**
+- Zero time is arbitrary but must be identical for all frames and corresponding messages (probably within one simulation).
+- Zero time does not need to coincide with the unix epoch.
+- It is recommended to use the start time of the simulation as zero time.
```

Comment:
```suggestion
- It is recommended to define the zero time as the actual clock time at the moment the simulation begins
```

I would make it more clear so everyone understands what is meant with a "start time"

Reply:
> I would make it more clear so everyone understands what is meant with a "start time"

I think however using "actual clock time" would make this unclearer, as this suggests that there is some relationship to wall clock time or something else that one would consider an actual clock (rather than a simulated one). Since we are not assuming a real-time simulation, talking about actual clocks seems problematic.

Maybe something like:

```suggestion
- It is recommended to define zero time to coincide with the start time (initial time step) of the simulation, i.e. simulation time zero.
```